pkg/collector/metrics.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2020 CaiCloud, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+var (
+	eventCount = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "kube_event",
+		Subsystem: "",
+		Name:      "count",
+		Help:      "Number of kubernetes event happened",
+	}, []string{"name", "involved_object_namespace", "namespace", "involved_object_name", "involved_object_kind", "reason", "type", "source"})
```

Comment:
`uid`? Wasn't that the whole point of the refector?

Reply:
We generally use the namespace/name  as a unique identifier instead of  uid due to its poor readability for the event.